inst/assets/pkgdown.js

```diff
@@ -39,6 +39,13 @@
     }
   });
 
+  $(function() {
```

Comment:
I don't think this needs to be in its own `function` block. You can move the following to the top after the headroom call. Note that I updated to refer to the `navbar` class. 

```js
$(document.body).css('padding-top', $('.navbar').height() + 10);
$(window).resize(function(){
  $(document.body).css('padding-top', $('.navbar').height() + 10);
});

```